### PR TITLE
nmap: update Polish translation

### PR DESCRIPTION
**Pull request by spageektti**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by spageektti** [comment]
I hide the TLDR Bot comments because I know that the page is outdated and it's still WIP.

**Review by Magrid0 — APPROVED** [review]
Looks good to me

**Review by blueskyson — APPROVED** [review]

**Comment by kbdharun** [comment]
Hi @spageektti is this PR open for review, or is there still some parts to update in the page?

**Comment by spageektti** [comment]
> Hi @spageektti is this PR open for review, or is there still some parts to update in the page?

Most of the page is still outdated. I will finish it at the end of this week.

**Comment by spageektti** [comment]
I thought this is up to date now

File: pages.pl/common/nmap.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
-`sudo nmap -sn {{ip_lub_nazwa_hosta}} {{opcjonalny_kolejny_adres_ip}}`
+`nmap -T5 -sn {{192.168.0.0/24|ip_lub_nazwa_hosta1,ip_lub_nazwa_hosta2,...}}`
 
-- Poza tym, uruchom domyśle skrypty, wykrywanie działających serwisów, OS fingerprinting oraz komendę traceroute:
+- Włącz wykrywanie systemu operacyjnego, wykrywanie wersji, skanowanie skryptów i traceroute hostów z pliku:
 
-`nmap -A {{adres_lub_adresy_ip}}`
+`sudo nmap -A -iL {{ścieżka/do/pliku.txt}}`
 
 - Skanuj podaną listę portów (użyj '-p-' dla wszystkich portów od 1 do 65535):
```
Comment (acuteenvy) [inline pages.pl/common/nmap.md:19]:
```suggestion
- Skanuj podaną listę portów (użyj `-p-` dla wszystkich portów od 1 do 65535):
```

File: pages.pl/common/nmap.md [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
 
-`nmap -p {{port1,port2,...,portN}} {{adres_lub_adresy_ip}}`
+`nmap -p {{port1,port2,...}} {{ip_lub_host1,ip_lub_host2,...}}`
 
-- Przeprowadź wykrywanie serwisów i ich wersji dla 1000 najczęstrzych portów używając domyślich skryptów NSE; Zapisz rezultat ('-oN') do pliku wyjściowego:
+- Przeprowadź wykrywanie usług i wersji dla 1000 najpopularniejszych portów używając domyślnych skryptów NSE, Zapisując wynik ('-oA') do plików wyjściowych:
```
Comment (acuteenvy) [inline pages.pl/common/nmap.md:23]:
```suggestion
- Przeprowadź wykrywanie usług i wersji dla 1000 najpopularniejszych portów używając domyślnych skryptów NSE, zapisując wynik (`-oA`) do plików wyjściowych:
```

File: pages.pl/common/nmap.md [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
 
-`nmap -sC -sV -oN {{top-1000-ports.txt}} {{adres_lub_adresy_ip}}`
+`nmap -sC -sV -oA {{top-1000-ports}} {{ip_lub_host1,ip_lub_host2,...}}`
 
-- Skanuj cel lub cele używając skryptów NSE 'default and safe':
+- Skanuj cel(e) ostrożnie używając skryptów NSE 'default and safe':
```
Comment (acuteenvy) [inline pages.pl/common/nmap.md:27]:
```suggestion
- Skanuj cel(e) ostrożnie używając skryptów NSE `default and safe`:
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**